wrappers/python/examples/pyglet_pointcloud_viewer.py

```diff
@@ -84,13 +84,25 @@ def rotation(self):
         Ry = rotation_matrix((0, 1, 0), math.radians(-self.yaw))
         return np.dot(Ry, Rx).astype(np.float32)
 
-
 state = AppState()
 
 # Configure streams
 pipeline = rs.pipeline()
 config = rs.config()
 
+pipeline_wrapper = rs.pipeline_wrapper(pipeline)
+pipeline_profile = config.resolve(pipeline_wrapper)
+device = pipeline_profile.get_device()
+
+found_rgb = False
+for s in device.sensors:
+    if s.get_info(rs.camera_info.name) == 'RGB Camera':
+        found_rgb = True
+        break
+if not found_rgb:
+    print("The connected device does not support RGB stream")
```

Comment:
~~The connected device does not support RGB stream~~ - " The demo requires Depth camera with Color sensor".